docs/modules/pkl-cli/pages/index.adoc

```diff
@@ -192,6 +193,33 @@ This should print something similar to:
 Pkl {pkl-version} (macOS 10.16, Java 11.0.9)
 ----
 
+=== Windows Installation
+Native Windows is not a currently supported platform for Pkl however you may be able to still achieve some level of success.  Official support is planned for the next release (targetting June timeframe).  Pkl does work on Windows through https://learn.microsoft.com/en-us/windows/wsl/install[WSL] but our Java Executables have some degree of compatibility with Windows native.
+
+The following instructions should work, but again no official support is provided:
+
+* You will need a Java Runtime installed (ie https://www.oracle.com/java/technologies/downloads/#jdk21-windows[Java SE / JDK 21] )
+* You need to download our JAR files for the CLI this can be found in our {uri-ci-artifacts}[Sonatype Repository] or from https://search.maven.org/remote_content?g=org.pkl-lang&a=pkl-cli-java&v=LATEST[Maven]
+* You can then run the Java Executable and see the following result:
+
+[source,shell]
+[subs="+attributes"]
+----
+> java -jar pkl-cli-java.jar --version
+Pkl 0.26.0-dev+21e0e14 (Windows 10 10.0, Java 21.0.2)
+----
+
+Note: You must use forward slashes in all paths for Windows with absolute paths prefixed with file:///.  The following examples are valid:
+[source,shell]
+[subs="+attributes"]
+----
+> java -jar pkl-cli-java.jar eval file:///C:/Code/pkl/test.pkl
+> java -jar pkl-cli-java.jar eval ../Code/pkl/test.pkl
+> java -jar pkl-cli-java.jar eval pkl/test.pkl
+----
+For more details on the current status of Windows support please see https://github.com/apple/pkl/issues/20[GitHub Issue #20].
```

Comment:
https://github.com/apple/pkl/issues/20[GitHub Issue #20] is used to track progress on support for the Windows platform.